Default.sublime-commands

```diff
@@ -215,5 +215,65 @@
     {
         "caption": "git: reset (reflog)",
         "command": "gs_reset_reflog"
+    },
+    {
+        "caption": "git flow: init",
```

Comment:
How would you feel about `flow: init` and so on for the rest of these commands?  The reason I ask is that I'd like not to clutter the command palette for non-users of Git flow.  Particularly with `init` and `pull` in there.